docs/_pages/typecheck.md

```diff
@@ -69,6 +69,57 @@ local a
 local b = nil
 ```
 
+## `unknown` type
```

Comment:
Can these be sub-headings? (three #s)

Reply:
Yeah.

Reply:
Done, and renamed `Primitive types` heading to `Builtin types`.